### PR TITLE
Add test for compatible (MM) hinting

### PR DESCRIPTION
**Pull request by khaledhosny**
Fixes https://github.com/adobe-type-tools/psautohint/issues/39

### CODE REVIEW

File: python/psautohint/__main__.py [hunk 1]
```diff
@@ -682,7 +682,9 @@ def get_options(args):
     )
     parsed_args = parser.parse_args(args)
 
-    if len(parsed_args.font_paths) > 1 and parsed_args.output_path:
+    if (parsed_args.output_paths and
+            len(parsed_args.font_paths) != len(parsed_args.output_paths)):
+        print(parsed_args)
```
Comment (miguelsousa) [inline python/psautohint/__main__.py]: is this a debug print?

File: python/psautohint/__main__.py [hunk 2]
```diff
@@ -682,7 +682,9 @@ def get_options(args):
...
         parser.error("-o/--output can't be used with multiple font inputs")
```
Comment (miguelsousa) [inline python/psautohint/__main__.py]: the check changed so the message should change accordingly

File: tests/test_hintfontsmm.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        self.outputPaths = outpaths
+        self.reference_font = reference
+        self.hintAll = True
+        self.verbose = False
+
+
+@pytest.mark.parametrize("base", glob.glob("data/*/*Masters"))
+def test_mmufo(base):
+    paths = sorted(glob.glob(base + "/*.ufo"))
+    options = Options(paths[0], paths[1:], [p + ".out" for p in paths[1:]])
```
Comment (miguelsousa) [inline tests/test_hintfontsmm.py]: better to define `reference`, `inpaths` and `outpaths` outside, I think

**Comment by miguelsousa** [comment]
please wrap the imports on this line as well https://github.com/adobe-type-tools/psautohint/pull/53/files#diff-df7531b6e02c6fd7b0cdebea307ebfb6R19

File: tests/test_hintfontsmm.py (hunk 3 above)
Comment (khaledhosny) [reply tests/test_hintfontsmm.py]: Naming variables is hard, I struggled with finding appropriate names so I avoided them altogether :) Using your suggestion now.

File: tests/test_hintfontsmm.py [hunk 4]
```diff
@@ -19,8 +19,12 @@ def __init__(self, reference, inpaths, outpaths):
 @pytest.mark.parametrize("base", glob.glob("data/*/*Masters"))
 def test_mmufo(base):
     paths = sorted(glob.glob(base + "/*.ufo"))
-    options = Options(paths[0], paths[1:], [p + ".out" for p in paths[1:]])
+    reference = paths[0]
+    inpaths = paths[1:]
+    outpaths = [p + ".out" for p in paths[1:]]
```
Comment (miguelsousa) [inline tests/test_hintfontsmm.py]: you can use `inpaths` here instead of `paths[1:]`

File: python/psautohint/__main__.py [hunk 5]
```diff
@@ -16,7 +16,8 @@
 from fontTools.misc.py23 import open
 
 from psautohint._psautohint import version as PSAUTOHINT_VERSION
-from psautohint.autohint import hintFiles, ACOptions, ACFontError, ACHintError, logMsg
```
Comment (miguelsousa) [inline python/psautohint/__main__.py]: wrap using parenthesis instead
```python
from psautohint.autohint import (hintFiles, ACOptions, ACFontError, ACHintError,
                                 logMsg)
```

Comment (khaledhosny) [reply python/psautohint/__main__.py]: Done.

Comment (miguelsousa) [reply python/psautohint/__main__.py]: 👍

**Pull request merged**